src/main/kotlin/com/sourcegraph/cody/history/HistoryTree.kt

```diff
@@ -1,84 +1,125 @@
 package com.sourcegraph.cody.history
```

Comment:
Can you make sure to move UI-only components to `ui` sub package?

Reply:
Fixed.

Reply:
FYI @mkondratek for context, we discussed today with Piotr about packaging strategy, and we concluded it's a little mess and we'll be putting UI-related code into `ui` subpackage for clear separation.